contracts/RelayRecipient.sol

```diff
@@ -92,8 +92,10 @@ contract RelayRecipient is RelayRecipientApi {
 	 * 			the contract may check the method-id for valid methods
 	 *  @param gas_price - the gas price for this transaction
 	 *  @param transaction_fee - the relay compensation (in %) for this transaction
+	 *  @param approval - first 65 bytes are reserved for the sender's signature, and the rest is
```

Comment:
first 65 bytes are checked by the RelayHub
(relayRecipient doesn't have to check them)

Reply:
Done